video/out/vo_gpu_next.c

```diff
@@ -77,6 +74,7 @@ struct priv {
     struct mp_log *log;
     struct mpv_global *global;
     struct ra_ctx *ra_ctx;
```

Comment:
This `ra_ctx` can probably be removed now?

Reply:
There are 24 uses of it left but the variable is indeed superflous, I could `s/p->ra_ctx/p->context->ra_ctx/` if you want.

Reply:
Ah, no need then. I'll probably get rid of it as part of refactoring those APIs into `gpu_next`.